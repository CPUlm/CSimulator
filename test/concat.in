6
000
11
101
01
101
11
111
00
000
00
111
11
