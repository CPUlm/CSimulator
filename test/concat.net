INPUT a,b
OUTPUT c
VAR
  a : 3, b : 2, c : 5
IN
c = CONCAT a b

