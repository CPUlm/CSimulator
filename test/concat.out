Step 1: 
=> c = 00011
Step 2: 
=> c = 10101
Step 3: 
=> c = 10111
Step 4: 
=> c = 11100
Step 5: 
=> c = 00000
Step 6: 
=> c = 11111
